### PR TITLE
add prettier-plugin-tailwindcss

### PR DESCRIPTION
**Pull request by rickyzhangca**
adding prettier-plugin-tailwindcss to auto sort tailwind classes

https://tailwindcss.com/blog/automatic-class-sorting-with-prettier

### CODE REVIEW

File: packages/prettier-config/config.js [hunk 1]
```diff
@@ -12,4 +12,5 @@ module.exports = {
   plugins: ["@trivago/prettier-plugin-sort-imports"],
   importOrder: ["<THIRD_PARTY_MODULES>", "^@/(.*)$", "^[./]"],
   importOrderSeparation: true,
+  "plugins": ["prettier-plugin-tailwindcss"]
```
Comment (rpmccarter) [inline packages/prettier-config/config.js:15]: can you just add this to line 12?

**Review by rpmccarter — CHANGES_REQUESTED** [review]
pls also bump the version in `package.json` - don't worry about semantic versioning, just bump it in some way

File: packages/prettier-config/config.js (hunk 1 above)
Comment (rickyzhangca) [reply packages/prettier-config/config.js:15]: how did i miss it

**Comment by hahnbeelee** [comment]
@rickyzhangca can ya give me ur npm username - we can add u

**Comment by rickyzhangca** [comment]
@hahnbeelee `rickyzhangca`, appreciated

**Pull request merged**